prefetch_db.go

```diff
@@ -200,7 +200,9 @@ func addDBfileToDB(urlDB string, repoName string) (MirrorDB, error) {
 		return MirrorDB{}, fmt.Errorf("url '%v' is invalid, cannot save it for prefetching", urlDB)
 	}
 	mirror := MirrorDB{urlDB, repoName, &now}
-	prefetchDB.Save(&mirror)
+	if db := prefetchDB.Save(&mirror); db.Error != nil {
+		return mirror, db.Error
```

Comment:
If an error happens it is usually nul value returned instead of valid structs like `mirror`.

Reply:
I kept it because this had been helpful in the past for identifying this issue with the db (mirror variable was fine, only the db failed). Anyway, returning `mirror` is confusing, I agree on that, I'll replace it with `MirrorDB{}`